README.rst

```diff
@@ -135,9 +117,24 @@ and use an *editable* installation of GT4Py:
 Dependencies for running tests locally and for linting and formatting
 source are listed in `requirements-dev.txt`.
 
+There are notebooks in the ``examples/`` directory that can be run using
```

Comment:
Need to reorganize this a bit. How does the user access examples if installed from pypi?

Reply:
I added a link to the examples folder in the github repo. It's not great but at least it's something, and github renders the notebooks decently